pkg/store/event/event.go

```diff
@@ -423,6 +423,16 @@ func (e *eventStore) getUID(_ context.Context, ns, name string) (string, error)
 	return UID, nil
 }
 
+// UpdateIncompleteEvents updates the incomplete event by the namespace and name
+func (e *eventStore) UpdateIncompleteEvents(_ context.Context, ns, name string) error {
+	return e.db.Model(core.Event{}).
+		Where(
+			"namespace = ? and experiment = ? and finish_time IS NULL",
```

Comment:
Do we need to handle the `Not found` error?

Reply:
I just tried it and there is no "not found error". I don't know if it is because update has no such errors.

Reply:
Got